nautobot/core/celery/log.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from celery import current_task
+
+
+class NautobotDatabaseHandler(logging.Handler):
+    """Custom logging handler to log messages to JobLogEntry database entries."""
+
+    def emit(self, record):
+        if current_task is None:
+            return
+
+        from nautobot.extras.models.jobs import JobResult
+
+        try:
+            self.format(record)
+
+            if record.task_id == "???":
+                return
+            job_result = JobResult.objects.filter(id=record.task_id)
+            if not job_result.exists():
+                return
+            job_result.first().log(
```

Comment:
I worry this will come back to bite us in the concurrency case. Are `task_id`s unique globally, even if two workers pick up the same task?

Reply:
The `task_id` is generated when enqueuing the job by celery's `apply_async` using `uuid()` so it has the same likelihood of collision that it had before when our `enqueue_job` was doing the same.